src/components/Editor/Editor.js

```diff
@@ -212,11 +211,9 @@ export default class Editor extends Component {
           <br />
           {'or create a new one'}
         </p>
-        <FlatButton
-          label='create'
-          onClick={() => this.setState({ newFuncOpen: true })}
-          icon={<FontIcon className='material-icons'>add</FontIcon>}
-        />
+        <a className='button button-primary' onClick={() => this.setState({ newFuncOpen: true })}>
```

Comment:
I prefer to have handler methods instead of setting a state directly in the JSX code. I usually keep the `render` method only for showing information.